linode-python/Pulumi.yaml

```diff
@@ -0,0 +1,8 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: python
+template:
+  description: A minimal Linode Python Pulumi program
+  config:
+    gcp:project:
```

Comment:
Replace with `linode:token`. See my comment for `linode-typescript/Pulumi.yaml`.